src/features/activityFeed/state/activityFeedActions.ts

```diff
@@ -14,15 +14,20 @@ export function loadDatasetLogs(qriRef: QriRef): ApiActionThunk {
 
 function fetchDatasetLogs(qriRef: QriRef): ApiAction {
   return {
-    type: 'dataset_logs',
+    type: 'dataset_activity',
     qriRef,
     [CALL_API]: {
       endpoint: `ds/activity`,
       method: 'POST',
       body: {
-        ref: `${qriRef.username}/${qriRef.name}`
+        ref: `${qriRef.username}/${qriRef.name}`,
+        pageSize: 100
+      },
+      query: {
+        pageSize: 100,
+        page: 2
```

Comment:
why are we fetching page 2?

Reply:
Ah, I meant to remove all of this, I was trying to get more than 25 results from this endpoint and left it in there.